### PR TITLE
[15.0][MIG] pos_fixed_discount

### PR DESCRIPTION
**Pull request by Rad0van**
Standard migration.

### CODE REVIEW

**Review by ivantodorovich — APPROVED** [review]
Could you check pre-commit please, seems red?

**Review by GabbasovDinar — APPROVED** [review]
Code LGTM!

File: pos_fixed_discount/static/src/js/FixedDiscountButton.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ odoo.define("pos_fixed_discount.FixedDiscountButton", function (require) {
                 startingValue: 0,
             });
             if (confirmed) {
-                var val = Math.round(Math.max(0, Math.min(100, parseFloat(payload))));
+                var val = parseFloat(payload);
```
Comment (chienandalu) [inline pos_fixed_discount/static/src/js/FixedDiscountButton.js:20]: What's the rationale for this change?

**Review by chienandalu — CHANGES_REQUESTED** [review]

File: pos_fixed_discount/static/src/js/FixedDiscountButton.js (hunk 1 above)
Comment (ivs-cetmix) [reply pos_fixed_discount/static/src/js/FixedDiscountButton.js:20]: Looks like it comes from https://github.com/OCA/pos/pull/888
Either it is needed to be merged first or we need to migrate "as is" and than apply this patch with a new PR

Comment (Rad0van) [reply pos_fixed_discount/static/src/js/FixedDiscountButton.js:20]: Yeah - it is from #888 I had it applied in local repo and forgot about it when doing the migration so the proper way should be as @isserver1 suggests...
